tools/pull_issues.sh

```diff
@@ -75,7 +81,7 @@ done
 cd ..
 
 # Check if filename has a slash (path). Move the file one directory up if it's just a filename.
-if [[ "${OUTFILE}" != *\/*  ]] || [[ "${OUTFILE}" != *\\* ]]
+if [[ "${OUTFILE}" != *\/*  ]] || [[ "${OUTFILE}" != *\\* ]]
```

Comment:
What is changed on this line?

Reply:
There was a missing space that raised an error.